tests/serve_request.rs

```diff
@@ -220,3 +221,23 @@ fn serves_requests_custom_index_notice(tmpdir: TempDir, port: u16) -> Result<(),
 
     Ok(())
 }
+
+#[rstest]
+fn serves_requests_with_path_prefix() -> Result<(), Error> {
+    let route = "foobar";
+    let server = server(&["--path-prefix", route]);
+    let client = Client::new();
+
+    let status = client.get(server.url()).send()?.status();
+
+    assert_eq!(status, StatusCode::NOT_FOUND);
```

Comment:
I'd appreciate a comment here about your expectation at this point in the test.

Reply:
Sorry, i don't get it. What do you mean?

Reply:
Basically, _why_ do you expect it to be `NOT_FOUND` there but `OK` later in the test? Give it a bit of a narrative. :)